src/linear_algebra/projective_space/subspace.lean

```diff
@@ -99,6 +89,10 @@ def gi : galois_insertion (span : set (ℙ K V) → subspace K V) coe :=
   le_l_u := λ S, subset_span _,
   choice_eq := λ _ _, rfl }
 
+/-- The span of a subspace is the subspace. -/
+@[simp] lemma span_coe (W : subspace K V) : span ↑W = W :=
+by { exact galois_insertion.l_u_eq gi W }
```

Comment:
You should never need `by exact` (they essentially cancel each other out). Also you can save a few characters by using dot notation here:
```suggestion
@[simp] lemma span_coe (W : subspace K V) : span ↑W = W := gi.l_u_eq W
```